examples/diff/index.html

```diff
@@ -83,20 +84,38 @@
       renderDiff();
     }
 
-    function keyboardInputHandler() {
+
+    /**
+     * Event handler for keyboard input.
+     * @param event The event object.
+     */
+    function keyboardInputHandler(event) {
       renderDiff();
       field.setValue(draftTextEl.value);
     }
 
+
+    /**
+     * Event handler triggered before the window unloads.
+     * @param event The event object.
+     */
+    function onBeforeUnloadHandler(event) {
+      window.removeEventListener('onbeforeunload', onBeforeUnloadHandler);
```

Comment:
Is this needed? I am curious since when the window is unloaded, it should logically release all the resources it had held and thus cleanup _all_ handlers for all events on the page right?